_pages/oidc/authorization.md

```diff
@@ -37,12 +37,11 @@ Correct Example
 ```ruby
 code_verifier = SecureRandom.hex
 => "5787d673fb784c90f0e309883241803d"
-code_challenge = Digest::SHA256.digest(code_verifier) # binary data
+code_challenge = Digest::SHA256.digest(code_verifier)
+=> "\xD4\x15)\xC7-\xFBJ\x82\x0F\x98\xAC=\xEB\x06\xDD\xE8\xC0\xDC\xBD\xAC\x8Ebm\xE9\xB5?\xB3m\xEE\x8B\xFF3" # binary digest
 url_safe_code_challenge = Base64.urlsafe_encode64(code_challenge)
-# RFC 4648 URL-safe Base64 encoding replaces "+" with "-" and "/" with "_" and trims trailing "="
-=> "1BUpxy37SoIPmKw96wbd6MDcvayOYm3ptT-zbe6L_zM"
-Base64.encode64(code_challenge) # wrong and URL-unsafe encoding
-=> "1BUpxy37SoIPmKw96wbd6MDcvayOYm3ptT+zbe6L/zM=" # wrong and URL-unsafe encoding
+# RFC 4648 URL-safe Base64 encoding replaces "+" with "-" and "/" with "_"
+=> "1BUpxy37SoIPmKw96wbd6MDcvayOYm3ptT-zbe6L_zM="
```

Comment:
[question] if we want to remove the trailing `=` we can do that by adding padding: `false` as an option to `Base64.urlsafe_encode64` but it was unclear to me if that was necessary. does anyone know offhand?

Reply:
I'm not sure if we need it or not, but since it works with it (based on my test integration I made forever ago, I'd be inclined to leave it in our docs.)

Reply:
it looks like we remove the padding if it exists, so it doesn't matter.

https://github.com/18F/identity-idp/blob/main/app/forms/openid_connect_token_form.rb#L191-L195